Azure/azure-cli
# [Extension] Turn on dynamic install by default

**Description**  

This PR modifies the default value of config option `extension.use_dynamic` from `no` to `yes_prompt`.

**Testing Guide**  

Run any extension command when the extension is not installed, you should be prompted to install the extension.

**History Notes**  


[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
